README.md

```diff
@@ -1 +1,143 @@
+# hrxp-api
```

Comment:
Can you add a TODO to finalize this doc?

Reply:
Kyle is going to take care of the documentation. He is going to integrate Swaggerhub into our repo sometime soon!